### PR TITLE
Fix login timeout on dump/restore

### PR DESCRIPTION
**Pull request by Aschen**
## What does this PR do?

Dump and restore index can be very long so the token will be periodically refreshed.

I had to all commands to add a `cleanInterval` after the command has run

### Other changes

 - catch error and continue to process other collection when an export/import fail

### CODE REVIEW

File: src/support/kuzzle.ts [hunk 1]
```diff
@@ -41,24 +43,33 @@ export class KuzzleSDK {
...
+  private refreshTimer?: NodeJS.Timeout;
 
   constructor(options: any) {
     this.host = options.host
     this.port = parseInt(options.port, 10)
     this.ssl = options.ssl || this.port === 443
     this.username = options.username
     this.password = options.password
-    this.loginTTL = options.loginTTL || '10s'
+    this.refreshLogin = options.refreshLogin || false
```
Comment (scottinet) [inline src/support/kuzzle.ts]: Why should this be an option?

As a user, I expect the CLI to take care of how the authentication token is created and refreshed.

If any command runs longer than the default login TTL (which is 90s), then I think that Kourou should automatically renew its token until the command ends.

Also, renewing every second or so seems a bit too often to me. If you have a login TTL of 90s, why not refreshing every 80s or so (10s to account for network latency seems largely enough)

What do you think?

File: src/support/kuzzle.ts [hunk 2]
```diff
@@ -68,8 +79,28 @@ export class KuzzleSDK {
...
+      }
+
+      log(chalk.green(`[ℹ] Loggued as ${this.username} (refreshLogin: ${this.refreshLogin}).`))
+    }
+  }
+
+  disconnect() {
+    this.sdk.disconnect()
+
+    if (this.refreshTimer) {
```
Comment (scottinet) [inline src/support/kuzzle.ts]: (nitpicking) the behavior of `clearInterval` on null or undefined vaues is a no-op, so this if-condition is unnecessary

File: src/support/kuzzle.ts (hunk 1 above)
Comment (Aschen) [reply src/support/kuzzle.ts]: Yes you're right, I'm gonna refresh it all the time if needed :+1:

File: src/support/kuzzle.ts (hunk 2 above)
Comment (Aschen) [reply src/support/kuzzle.ts]: I can't remove the condition otherwise Typescript will complain :expressionless:

**Pull request merged**